hazelcast-client/src/main/java/com/hazelcast/client/console/ClientConsoleApp.java

```diff
@@ -1106,9 +1108,9 @@ protected void handleContains(String[] args) {
         String iteratorStr = args[0];
         boolean key = false;
         boolean value = false;
-        if (iteratorStr.toLowerCase().endsWith("key")) {
+        if (iteratorStr.toLowerCase(StringUtil.LOCALE_INTERNAL).endsWith("key")) {
```

Comment:
Here we could also safe a double `toLowerCase()` call if it's a value (in case the JVM doesn't optimize this for us).
```
String iteratorType = lowerCaseInteral(iteratorStr);
if (iteratorType.endsWith("key")) {
   ...
} else if (iteratorType.endsWith("value")) {
   ...
}
```

Reply:
+1 I'll update it in both console apps.